Formula/nuvie.rb

```diff
@@ -51,4 +51,9 @@ def caveats; <<~EOS
       #{var}/nuvie/nuvie.cfg
   EOS
   end
+
+  test do
+    assert_predicate bin/"nuvie", :exist?
+    assert_predicate bin/"nuvie", :executable?
+  end
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook and neither is just checking if a file exists.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thank you for the feedback! @SMillerDev 
I am working on improving these tests, and will update you once they're done. :)

Reply:
I was unable to find any command line only functions for Nuvie, hence I've made the tests similar to the tests in `bastet` which require running and then killing the program. Please let me know if any other tests should be added.

Thanks!